README.md

```diff
@@ -32,3 +32,13 @@ By default the following are allowed:
 
 By default, guests will not be able to list other users in the system, if a guest user gets added to a group he will be able
 to list users within that group (and, for example, share files with those users). 
+
+## Auto-convert guest users into SAML users
```

Comment:
```suggestion
## Auto-convert guest users into system users
```

it doesn't have to be SAML but works with any user back-end ;)

Reply:
How about `full users` instead of `system users`?

Reply:
Fine by me :)